spec/reek/smells/too_many_instance_variables_spec.rb

```diff
@@ -10,12 +10,26 @@
 
   it_should_behave_like 'SmellDetector'
 
+  def default_max_ivars
+    Reek::Smells::TooManyInstanceVariables::DEFAULT_MAX_IVARS
+  end
+
+  def too_much_ivars
+    default_max_ivars + 1
+  end
```

Comment:
I’m aware I’m bordering on [obsessing over grammar minutiæ](https://www.youtube.com/watch?v=UJPeg1b-jVo), but let’s go with `too_many_ivars` for countable amounts. :)